config/admin-cli.properties

```diff
@@ -19,11 +19,43 @@ cli.controller.grpc.uri=tcp://localhost:9090
 cli.store.metadata.backend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
-pravegaservice.cluster.name=pravega/pravega
+pravegaservice.clusterName=pravega/pravega
```

Comment:
@ManishKumarKeshri please, use the existing name structure, as I think that the property name used in this PR is from the old naming structure. @ravisharda is that correct?

Reply:
It should be `clusterName` as mentioned here: https://github.com/pravega/pravega/blob/b5a809388a650e95ec2cd901a2b2f92af47b7081/segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java#L54

Reply:
Got it, missed that one.